Code/Framework/AzQtComponents/AzQtComponents/Components/DockBarButton.cpp

```diff
@@ -15,6 +15,7 @@
 #include <QStyleOptionToolButton>
 #include <QStylePainter>
 #include <QTimer>
```

Comment:
Nitpick: Since you're removing the `QTimer::singleShot` you can remove the `QTimer` include as well